sources/standbycontrol.cpp

```diff
@@ -179,16 +179,20 @@ void StandbyControl::wakeup() {
 
             m_wifiControl->on();
 
+            QObject *context = new QObject(this);
+            QObject::connect(m_wifiControl, &WifiControl::connected, context, [=]() {
```

Comment:
I think it would be better to permanently hook up both `WifiControl::disconnected` and `WifiControl:connected` signals to the integrations during setup.
This would also cover other WiFi disconnect issues like going out of range, or getting disconnected for other reasons.

Reply:
Another issue is that it's not guaranteed that the `WifiControl::connected` signal is ever fired after this point!
Wifi might already be connected and the connected signal emitted. This would then leave the integrations disconnected...
Therefore, we need to check if Wifi is already connected:
- yes: connect integrations
- no: delayed connect with signal